watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination.py

```diff
@@ -351,89 +935,540 @@ def LCOT(b):
             * b.zo_costing.utilization_factor
         )
 
+    @m.fs.Expression(
+        doc="Levelized cost of treatment with respect to volumetric feed flow, not including externalities"
+    )
+    def LCOT_wo_revenue(b):
+        return (
+            b.total_capital_cost * b.zo_costing.capital_recovery_factor
+            + b.total_operating_cost
+        ) / (
+            pyunits.convert(
+                b.feed.properties[0].flow_vol,
+                to_units=pyunits.m**3 / pyunits.year,
+            )
+            * b.zo_costing.utilization_factor
+        )
+
+    @m.fs.Expression(
+        doc="Levelized cost of water with respect to volumetric permeate flow"
+    )
+    def LCOW(b):
```

Comment:
This is likely something we can revisit since this was put in place before our costing framework was refined. I would think this could be trimmed down using functionality from our costing framework.

Reply:
Yea, I can address this in a subsequent PR.